api/mesh/v1alpha1/envoy.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package kuma.mesh.v1alpha1;
+
+option go_package = "github.com/kumahq/kuma/api/mesh/v1alpha1";
+
+message EnvoyConfiguration {
+  enum XdsTransportProtocolVariant {
+    // This value defines default control-plane variant.
```

Comment:
Can we enrich this sentence?
I mean `This value defines default control-plane variant, it depends on the CP DeltaXds switch to enable incremental xDS or just enable state of the world xDS` some stuff like this, otherwise we don't know the default behavior at first glance.

Reply:
All right, maybe in the future once we'll remove the `KUMA_EXPERIMENTAL_DELTA_XDS` and just enable the deltaXDS by default, so there's no options for the CP but just configured by the DP's exclusive options.